tests/sync/sync_manager.cpp

```diff
@@ -78,6 +78,7 @@ TEST_CASE("sync_manager: `path_for_realm` API", "[sync]") {
     const std::string identity = "foobarbaz";
     auto user = SyncManager::shared().get_user(identity, ENCODE_FAKE_JWT("dummy_token"), ENCODE_FAKE_JWT("not_a_real_token"), auth_server_url, dummy_device_id);
     auto local_identity = user->local_identity();
+    auto server_identity = user->identity();
```

Comment:
I think you'd also want to check `REQUIRE(server_identity == local_identity)` here?

Reply:
They shouldn't have the same value? 🤔

Reply:
Actually there is no need to keep local_identity